api/envoy/extensions/filters/http/rate_limit_quota/v3/rate_limit_quota.proto

```diff
@@ -382,11 +377,10 @@ message RateLimitQuotaBucketSettings {
   // The interval at which the data plane (RLQS client) is to report quota usage for this bucket.
   //
   // When the first request is matched to a bucket with no assignment, the data plane is to report
-  // the request immediately in the :ref:`RateLimitQuotaUsageReports
```

Comment:
This might be out of the context of this PR but wasn't sure the best place to leave this. In our use case, the RQLS management server needs to know the Envoy node information in order to understand where the usage report is coming from. Should the first `RateLimitQuotaUsageReports` that Envoy sends contain the `Node` similar to other xDS APIs? Thanks!

Reply:
Thanks for the question!

I created an issue for this and we can discuss further over there. Resolving the conversation here....